pkg/apis/v1alpha1/securitypolicy_types.go

```diff
@@ -110,31 +110,9 @@ type SecurityPolicyPort struct {
 // SecurityPolicyStatus defines the observed state of SecurityPolicy.
 type SecurityPolicyStatus struct {
 	// Conditions describes current state of security policy.
-	Conditions []SecurityPolicyCondition `json:"conditions"`
+	Conditions []Condition `json:"conditions"`
 }
 
-// SecurityPolicyCondition defines the condition of policy.
-type SecurityPolicyCondition struct {
-	// Type defines the type of condition.
-	Type SecurityPolicyStatusCondition `json:"type"`
-	// Status shows the status of condition, one of True or False.
-	Status corev1.ConditionStatus `json:"status"`
-	// Reason shows a brief reason of condition.
-	Reason string `json:"reason,omitempty"`
-	// Message shows a human readable message about the condition.
-	Message string `json:"message,omitempty"`
-}
-
-// SecurityPolicyStatusCondition is an aspect of operator state.
-type SecurityPolicyStatusCondition string
-
-const (
-	// Ready indicates whether NSX Operator was able to successfully make
-	// a PATCH call to create/update Security Policy on NSX associated
-	// with the user created Security Policy CR
-	SecurityPolicyReady SecurityPolicyStatusCondition = "Ready"
```

Comment:
SecurityPolicyReady  is used in securitypolicy_controller.go, if removed here, need to change that file.

Reply:
My bad, didn't notice that we already have controller code for securitypolicy CRD. Updated `securitypolicy_controller.go` and `securitypolicy_controller_test.go`, thanks for reminder.